API.md

```diff
@@ -0,0 +1,76 @@
+## Classes
+
+<dl>
+<dt><a href="#AsyncAPIDiff">AsyncAPIDiff</a></dt>
+<dd><p>Implements methods to deal with diff output.</p>
+</dd>
+</dl>
+
+## Functions
+
+<dl>
+<dt><a href="#diff">diff(firstDocument, secondDocument, config)</a> ⇒ <code><a href="#AsyncAPIDiff">AsyncAPIDiff</a></code></dt>
+<dd><p>Generates diff between two AsyncAPI documents</p>
+</dd>
+</dl>
+
+<a name="AsyncAPIDiff"></a>
+
+## AsyncAPIDiff
+Implements methods to deal with diff output.
+
+**Kind**: global class  
+
+* [AsyncAPIDiff](#AsyncAPIDiff)
+    * [.breaking()](#AsyncAPIDiff+breaking) ⇒ <code>Array.&lt;DiffOutputItem&gt;</code>
+    * [.nonBreaking()](#AsyncAPIDiff+nonBreaking) ⇒ <code>Array.&lt;DiffOutputItem&gt;</code>
+    * [.unclassified()](#AsyncAPIDiff+unclassified) ⇒ <code>Array.&lt;DiffOutputItem&gt;</code>
+    * [.getOutput()](#AsyncAPIDiff+getOutput) ⇒ <code>Output</code>
+
+<a name="AsyncAPIDiff+breaking"></a>
+
+### asyncAPIDiff.breaking() ⇒ <code>Array.&lt;DiffOutputItem&gt;</code>
```

Comment:
imho types like `DiffOutputItem` should be in API docs too

Reply:
Is there a way to include the TypeScript types when `jsdoc2md` runs? I'm checking for some solutions now.

Reply:
jsdoc2md operates on js files, not ts, so I bet you need to somehow annotate types, maybe with `@type`, I think @jonaslagoni used it. It can also be done as a followup if you cannot find an easy solution now, no worries

Reply:
You might be able to do something along the lines of:

```
/**
 * Some description
 * @typedef {Object} DiffOutputItem
 * @property {string} ...
 */
type DiffOutputItem {}
```

Reply:
cool, I will try that. Thanks :)

Reply:
yeah, this doesn't work. Main problem is since there are no actual types in JS, the compiled output file that contains all the types is pretty much empty. So, JSDoc2md doesn't have any type output to work with.

Reply:
@derberg @jonaslagoni Can we merge this PR? Since this problem is still there, I think we can work on this in the future PRs. 
This PR adds documentation so I think it should be merged. And then I can work on solving this problem :)